src/vs/editor/common/commonCodeEditor.ts

```diff
@@ -119,7 +120,8 @@ export abstract class CommonCodeEditor extends Disposable implements editorCommo
 		domElement: IContextKeyServiceTarget,
 		options: editorOptions.IEditorOptions,
 		instantiationService: IInstantiationService,
-		contextKeyService: IContextKeyService
+		contextKeyService: IContextKeyService,
+		clipboardService: IClipboardService
```

Comment:
There's no need for the `IClipboardService` in this class, so it should be removed from here.